torch/_inductor/codegen/cpp.py

```diff
@@ -151,6 +151,10 @@ def float16_reduction_prefix(rtype):
 
 
 def parallel_num_threads():
+    # Current platform does not support OMP
+    if codecache.is_omp_enabled:
+        return 1
+
     threads = config.cpp.threads
```

Comment:
assert config says 1? or default? I would hate for folks to tweak the config thinking something changed, only to have it get early returned by no omp